ssl/record/rec_layer_s3.c

```diff
@@ -337,9 +337,9 @@ int ssl3_write_bytes(SSL *s, int type, const void *buf_, size_t len,
 {
     const unsigned char *buf = buf_;
     size_t tot;
-    size_t n, split_send_fragment, maxpipes;
+    size_t n, max_send_fragment, split_send_fragment, maxpipes;
 #if !defined(OPENSSL_NO_MULTIBLOCK) && EVP_CIPH_FLAG_TLS1_1_MULTIBLOCK
-    size_t max_send_fragment, nw;
+    size_t nw;
```

Comment:
Yes it is, as now `max_send_fragment` is also use outside the  
`#if !defined(OPENSSL_NO_MULTIBLOCK)...`